src/resources/styles.ts

```diff
@@ -110,6 +110,11 @@ export const haStyle = css`
     color: var(--app-header-text-color, white);
   }
 
+  app-toolbar,
+  div[sticky] {
```

Comment:
This style is too generic and should be more specific.

Reply:
How so? Both should always have that height or is it just the `div[sticky]` that is the issue? If that's the case, could do something like:

```
  app-toolbar,
  app-header div[sticky] {
    height: 48px;
  }
```

Reply:
Added the above